help/de/Help_ch_Account-Actions.xml

```diff
@@ -477,7 +418,8 @@
       </listitem>
 
       <listitem>
-        <para>Gibt es einen Anfangsbestand?
+        <para>Gibt es einen Anfangsbestand oder ist dies gar das Gegenkonto für ihre
```

Comment:
#: gnucash/gtkbuilder/dialog-account.glade:1517
msgid ""
"This account holds opening balance transactions. Only one account per "
"commodity can hold opening balance transactions."
msgstr ""
"Dieses Konto enthält die Anfangsbestand-Buchungen. Pro Währung/Aktie kann "
"nur ein Konto die Anfangsbestand-Buchungen enthalten."

Reply:
ok.